### PR TITLE
Add an spy menu component #446

### PR DESCRIPTION
**Pull request by carlosra85**
#446

## How Has This Been Tested

Unit test

## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] I have read the **CONTRIBUTING** document
- [x] My code follows the code style of this project
- [x] My change requires a change to the documentation 
- [x] I have updated the documentation accordingly (README.md for each UI component)
- [x] I have added tests to cover my changes (at least 1 spec for each UI component with the same coverage as the master branch)
- [x] All new and existing tests passed
- [ ] A new branch needs to be created from master to evolve previous versions
- [ ] Increase version in package.json following [Semantic Versioning](https://semver.org/)
- [x] All UI components must be added into the showcase (at least 1 component with the default settings)
- [x] Add the issue into the right [project](https://github.com/systelab/systelab-components/projects) with the proper status (In progress)

### CODE REVIEW

**Review by oGalceran1 — DISMISSED** [review]
Very interesting component. Thanks

File: projects/showcase/src/app/components/spy-menu/showcase-spy-menu-dialog.component.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+		this.menuItems.push(spyMenuItem1);
+		this.menuItems.push(spyMenuItem2);
+		this.menuItems.push(spyMenuItemDisabled);
+		this.menuItems.push(spyMenuItemHidden);
+		this.menuItems.push(spyMenuItem3);
+
+		this.sectionSelected = 'section1';
+	}
+
+	public onSectionChange(section: string) {
```
Comment (vinyulis) [inline projects/showcase/src/app/components/spy-menu/showcase-spy-menu-dialog.component.ts:49]: :void

**Review by vinyulis — APPROVED** [review]
I don't know if I would had done a full component with scrollspy and the spymenu embedded and ng-content inside... But the solution is very nice!. Thx!!!

**Review by fjaguado — DISMISSED** [review]

**Review by oGalceran1 — APPROVED** [review]

**Review by vinyulis — APPROVED** [review]

**Review by alfonsserra — APPROVED** [review]
Great!!!!

**Pull request merged**